### ISSUE TITLE
Error loading any of the sections of the main menu

### ISSUE THREAD

**Issue by kordys** [Kentico/KInspector#34]
I'm getting an error message when accessing any of the sections of the main menu:
```
Could not load file or assembly 'file:///c:\KInspector\KInspector\Ninject.Extensions.Conventions.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
```
The file path seems to be correct, no special file system permissions set.

*Note: I didn't find a way how to add the BUG label.*

**Comment by andrewradburnd** [comment]
Hey Kordys,

I had this issue aswell for a work around I had to right click on each dll and exe in the KInspector directory and click Unblock. 

Right Click > Properties > General Tab > Security Section > Unblock

![screen shot 2015-08-07 at 12 06 34 pm](https://cloud.githubusercontent.com/assets/2239329/9129516/23c19658-3d1c-11e5-8d56-b11468fb7442.png)

**Comment by kordys** [comment]
Hi Andrew,

thanks for the tip! It works for me as well. You don't need to go through all the dlls and exe file. Just unblock the downloaded zip package and unblock the package. This makes the trick. Anyway it would be great to fix this issue or at least mention this step in the docu/instructions.

**Comment by andrewradburnd** [comment]
No problem, I came across the issue about 30 mins before you posted your question. Knowing you can do it on the zip file as well makes life easier!

**Comment by petrsvihlik** [comment]
Should be possible to solve by adding loadFromRemoteSources to the app.config:
http://through-the-interface.typepad.com/through_the_interface/2011/07/loading-blocked-and-network-hosted-assemblies-with-net-4.html

**Comment by ondrejsevcik** [comment]
Fixed in #39
